Tools/build/regen-configure.sh

```diff
@@ -6,18 +6,19 @@ set -e -x
 # sync with this script. Use the same container image than the job so the job
 # doesn't need to run autoreconf in a container.
 IMAGE="ghcr.io/python/autoconf:2025.01.02.12581854023"
+# shellcheck disable=SC2034
```

Comment:
Why disable this one? Does Ruff check .sh files?

Reply:
Ah no that's me runbning shellcheck. It's just that it complains that it's not used.

Reply:
I've reverted this, I think it should be in a distinct PR from `ruff` changes.